s3cmd

```diff
@@ -2047,6 +2049,20 @@ def cmd_setacl(args):
         update_acl(s3, uri, seq_label)
     return EX_OK
 
+def cmd_setversioning(args):
+    cfg = Config()
+    s3 = S3(cfg)
+    uri = S3Uri(args[0])
```

Comment:
This should work only on a bucket, so I think that it would be nice to add a check to avoid mistakes/confusions:
```
    if bucket_uri.object():
        raise ParameterError("Only bucket name is required for [setversioning] command")
```